### ISSUE TITLE
IE 10 Member not found.

### ISSUE THREAD

**Issue by igorushko** [facebook/react#7320]
I catch Member not found exception on react 15.2.1 and ie10

It is on input onChange in SyntheticEvent on event.cancelBubble = true; line

``` javascript
stopPropagation: function () {
        var event = this.nativeEvent;
        if (!event) {
          return;
        }
        if (event.stopPropagation) {
          event.stopPropagation();
        } else {
          event.cancelBubble = true;
        }
        this.isPropagationStopped = emptyFunction.thatReturnsTrue;
      }
```

**Comment by nhunzaker** [comment]
This isn't an issue on master, but I believe it's because of a change to controlled inputs here: https://github.com/facebook/react/commit/045f1a791c6e17253e9d927ffca70ae5d00b4fe5. I could be wrong, but I think this is because it avoids creating a custom change event (But really, I don't know).

[This occurs when dispatching custom events that don't bubble (I believe) in IE](http://stackoverflow.com/questions/7993085/ie-9-showing-displaying-member-not-found). There are a couple of work arounds, one is to perform a check like `typeof event.cancelBubble !== 'unknown'`. This is mentioned in the jQuery issue tracker:

https://bugs.jquery.com/ticket/10004

My intuition here says that this would some how hinder bubbling. But it does not! I tested this against IE9-11, Chrome, Firefox, and Safari.

I've got a PR here with the details:

https://github.com/facebook/react/pull/7343
